bootloader/start64.c

```diff
@@ -10,6 +10,20 @@ void memcpy (uint64_t dest, uint64_t src, uint64_t n) {
   }
 }
 
+void memmove (uint64_t dest, uint64_t src, uint64_t n) {
```

Comment:
The distinction between memcpy and memmove is historical technical debt. Just only having memmove will make your code easier.